engine/src/test/java/com/redhat/cloud/notifications/TestLifecycleManager.java

```diff
@@ -0,0 +1,125 @@
+package com.redhat.cloud.notifications;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import io.smallrye.reactive.messaging.connectors.InMemoryConnector;
+import org.postgresql.ds.PGSimpleDataSource;
+import org.testcontainers.containers.MockServerContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.lang.reflect.Field;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.redhat.cloud.notifications.events.EventConsumer.INGRESS_CHANNEL;
+import static com.redhat.cloud.notifications.events.FromCamelHistoryFiller.EGRESS_CHANNEL;
+import static com.redhat.cloud.notifications.events.FromCamelHistoryFiller.FROMCAMEL_CHANNEL;
+import static com.redhat.cloud.notifications.processors.camel.CamelTypeProcessor.TOCAMEL_CHANNEL;
+import static com.redhat.cloud.notifications.processors.email.EmailSubscriptionTypeProcessor.AGGREGATION_CHANNEL;
+
+public class TestLifecycleManager implements QuarkusTestResourceLifecycleManager {
```

Comment:
Copied from `backend/.../TestLifecycleManager` with only 2 lines of code changed (properties).